.buildkite/premerge.steps.yaml

```diff
@@ -81,3 +81,11 @@ steps:
     if: (build.env("SLACK_NOTIFY") == "true" || build.branch == "master") && build.env("SLACK_NOTIFY") != "false" && build.env("ENGINE_NET_TEST") != "true"
     commands: "powershell ./ci/build-and-send-slack-notification.ps1"
     <<: *windows
+
+notify:
+  - slack: 
+      channels: 
+        - "#unreal-gdk-builds"
+      message: "<!subteam^SDB5161M5> Build failure! Please triage and post ticket." # SDB5161M5 is @support-unreal
+    if: build.env("BUILDKITE_TRIGGERED_FROM_BUILD_PIPELINE_SLUG") != "unrealgdktestgyms-premerge" && (build.state == "failed" || build.state == "canceled") && build.branch == "master"
```

Comment:
Do we also want to alert on cancelled? I would assume this can only happen with human intervention

Reply:
Yeeeeee. Because then a user can cancel a build for master and then we don't have a proper confirmation it works

Reply:
Hmmm, I think if someone cancels a build they usually know what they are doing, but I agree that this is "safer".